### DISCUSSION TITLE
[request] Traffic Replication

### DISCUSSION THREAD

**Discussion by montanaflynn** [Kong/kong#7351] [Ideas and feature requests]
Kong could send requests to multiple upstream targets for testing services and machines without effecting end-user experience. Here's a [similar project](https://github.com/buger/gor) for more information on traffic replay and inspiration for the capabilities and options that the plugin might use.

**Comment by lucamaraschi** [comment]
:+1:

**Comment by sonicaghi** [comment]
related to #210
